app/helpers/search_helper.rb

```diff
@@ -46,7 +46,11 @@ def sort_by_link(title, sort_key, query, record_type)
 
     params['sort_key'] = sort_key
 
-    url = "/search?#{params.to_query}"
+    url = if specified_url
```

Comment:
I'm curious what the convention is here:
if specified_url or unless specified_url.nil?   I think the latter but I'm not sure

Reply:
I think this is fine because nil or false will be falsey